config/routes.rb

```diff
@@ -3,6 +3,12 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  root 'users#index'
+
+  resources :users do 
+     resources :posts, only: [:index, :show]
+      post '/posts', to: 'posts#index'
+    # get '/posts/:id', to: 'posts#show'
```

Comment:
Kindly remove the commented piece of code to keep the code base clean.

Reply:
Done, as advised